Dart/src/com/jetbrains/lang/dart/ide/actions/DartEditorNotificationsProvider.java

```diff
@@ -50,6 +51,12 @@ public EditorNotificationPanel createNotificationPanel(@NotNull final VirtualFil
     }
 
     if (PubspecYamlUtil.PUBSPEC_YAML.equalsIgnoreCase(vFile.getName())) {
+
+      // Defer to the Flutter plugin for package management and SDK configuration if appropriate.
+      if (FlutterUtil.isFlutterPluginInstalled() && FlutterUtil.hasFlutterModule(myProject)) {
+        return null;
```

Comment:
You disable notification wihtout checking current module type. Is this what you intended to do?